python/tests/unit/arcticdb/version_store/test_deletion.py

```diff
@@ -30,15 +27,15 @@ def test_delete_version_with_update(version_store_factory, pos, sym):
     symbol = sym
 
     idx = pd.date_range("1970-01-01", periods=100, freq="D")
-    df = pd.DataFrame({"a": range(len(idx))}, index=idx)
+    df = pd.DataFrame({"a": range(len(idx))}, index=idx, dtype=np.int64)
```

Comment:
A bit worried about why this change is necessary.

Is this a behavioural change in Pandas that we should actually support?